agrirouter-api-java-api/src/test/java/com/dke/data/agrirouter/api/factories/impl/ReadRouterDeviceTest.java

```diff
@@ -0,0 +1,29 @@
+package com.dke.data.agrirouter.api.factories.impl;
+
+import com.google.gson.Gson;
+import de.dev4agriculture.dto.RouterDevice;
+import org.junit.jupiter.api.Test;
+
+public class ReadRouterDeviceTest {
+
+  @Test
+  void givenValidRouterDeviceCanBeCreated() {
```

Comment:
A test without assertion is not a test. Add an meaningful assertion or remove the test.

Reply:
Removed